components/page/index.js

```diff
@@ -5,10 +5,10 @@ import { Footer } from '@components/footer'
 
 import { StyledPage } from './styled'
 
-const Page = ({ isErrorPage, children, admin, wrap, ...rest }) => (
+const Page = ({ isErrorPage, children, admin, wrap, innerPadding = [2, 0], ...rest }) => (
```

Comment:
I know you said you needed to add this, but I am not exactly sure why. You should be able to setup padding on the page you need to with an element inside of `Page`, like a `Flex` or `Box`.

I'm also worried that this may have unintended side-effects on all the pages that currently use `Page`, although it's probably fine.

Reply:
Add this because in the designs the cards go full width to the edge of the page

![image](https://user-images.githubusercontent.com/1618764/65317249-c4391580-db9b-11e9-851e-c6502bc94e83.png)